src/Stripe.net/Infrastructure/JsonConverters/StripeExternalAccountConverter.cs

```diff
@@ -21,20 +21,26 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
 
```

Comment:
Yeah, this wasn't being used that's why I refactored, but if we go with Source everywhere, we don't need that either.